app_flutter/lib/commit_box.dart

```diff
@@ -132,7 +132,7 @@ class CommitOverlayContents extends StatelessWidget {
                       onPressed: () async {
                         final String githubUrl =
                             'https://github.com/${commit.repository}/commit/${commit.sha}';
-                        await launch(githubUrl);
+                        launch(githubUrl);
```

Comment:
Did you need to await this future for whatever reason?

Reply:
`url_launcher` example did it, but here we really don't care about when the url opens.